### DISCUSSION TITLE
Link to Twitch homepage in /mentions menu

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#5267] [Ideas]
I suggest, by analogy with other tabs, to add a link to twitch.tv in /mentions menu to simplify the way there from Chatterino.

There is little point in explaining the task in detail, because it looks easy and if a brief explanation is not enough, then there is no point in wasting time on detailed explanations and implementation.

**Comment by Nerixyz** [comment]
> There is little point in explaining the task in detail

I agree, the task itself is easy, but I don't really see _why_ one would go to the Twitch homepage from the `/mentions` channel (you can already open a usercard and click on the profile picture of the user that mentioned you). I feel like it would fit better for the `/whispers` channel. Which use-case am I missing?

**Reply by x9136** [reply]
> Which use-case am I missing?

Imagine that after booting up your computer, you start Chatterino. It regularly opens in the /mentions tab. The channels on which you write in chat are live in the evening, so the interest is higher in channels for background viewing. And so on.

This little thing would make it more convenient to open Twitch on the right page without first opening a browser or other tabs.

> I feel like it would fit better for the /whispers channel.

There's already a proper link for the /whisper tab. Also, I find it hard to imagine the /whisper tab being as popular as /mentions, so the link to Twitch is likely to be many times more useless on the /whisper tab than on the /mentions tab.

**Reply by pajlada** [reply]
> There's already a proper link for the /whisper tab. Also, I find it hard to imagine the /whisper tab being as popular as /mentions, so the link to Twitch is likely to be many times more useless on the /whisper tab than on the /mentions tab.

They are not analogous.
`/whispers` has a link in its menu to open your whispers on Twitch.
There's no equivalent for `/mentions` on Twitch - if there is we can add a link to that from the `/mentions` channel.

**Reply by x9136** [reply]
I was talking about habits. /mentions is the only tab where you can go to the menu for a link and not find it there.
